### PR TITLE
[Gradle] Update to provide Android Studio 3 support

### PR DESCRIPTION
**Pull request by lernerb**
* Update the README
* Update gradle to 3.X
* Remove old tests relying on old version of Robolectric

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (ronshapiro) [inline gradle/wrapper/gradle-wrapper.properties:6]: There are a few versions after this one, should the update at least go to the latest?

File: library/src/test/java/com/venmo/view/TooltipViewTest.java [hunk 2]
```diff
@@ -1,68 +0,0 @@
...
-
-    private static final @IdRes int sampleId = 234;
-
-    private TooltipView createView() {
-        Context context = Robolectric.buildActivity(Activity.class).create().get();
-        return new TooltipView(context);
-    }
-
-    @Test
-    public void settersInvalidateView() {
```
Comment (ronshapiro) [inline library/src/test/java/com/venmo/view/TooltipViewTest.java:31]: Why are you deleting this test?

**Review by ronshapiro — CHANGES_REQUESTED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (lernerb) [reply gradle/wrapper/gradle-wrapper.properties:6]: Yeah will do - I can bump everything to the latest and greatest.

File: library/src/test/java/com/venmo/view/TooltipViewTest.java (hunk 2 above)
Comment (lernerb) [reply library/src/test/java/com/venmo/view/TooltipViewTest.java:31]: I tried updating to both 3.1 and 3.8 of robolectric and it's spewing a resource error still, will try and resolve it and will revert.

**Comment by lernerb** [comment]
@ronshapiro Fixed both issues, gonna re-open a new PR since it's such a small change and I don't feel like reverting and rebasing since it's on the remote server already.

**Pull request closed without merging**